helpers/colour-contrast/rgb.js

```diff
@@ -0,0 +1,270 @@
+/**
+ * A class to parse color values
+ * @author Stoyan Stefanov <sstoo@gmail.com>
+ * @link   http://www.phpied.com/rgb-color-parser-in-javascript/
+ * @license MIT license
+ */
```

Comment:
Thoughts about simply not supporting non-RGB colors with this library? This is a lot of code to need to include just to support the color strings. If places need this, I think they could always call `getComputedStyle` on themselves to convert whatever they have to RGB anyway...

Reply:
On one hand I agree that it's probably not necessary, on the other hand removing that turns this library into a pretty simple string parser with added steps. I half like the idea of this library working with any kind of colour that it could be provided (except for something with an alpha channel). I'm not sure, I'll defer to you.

Reply:
I'm leaning towards only RGB... there's still lots of value here. For example, the [WebAIM Checker](https://webaim.org/resources/contrastchecker/) only supports RGB, as [does this one](https://contrastchecker.com/) and [this one](http://colorsafe.co/) and [this one](https://snook.ca/technical/colour_contrast/colour.html#fg=33FF33,bg=333333).

I can't think of any places in our system where folks are inputing colours that aren't in RGB, including our colour picker component in MVC.